docs/topics_5/web-adding-global-custom-labels.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_labels_{context}']
+= Adding global custom labels
+
+Global labels apply to all projects and are automatically applied to analyses for which they fit the definitions of a project.
+
+The {ProductShortName} includes a preconfigured set of global labels and you can define your own custom global labels.
+
+These labels are displayed and defined on the *Labels configuration* screen,
+contains two tabs: *System labels* and *Custom labels*. *System labels* contains a list of preconfigured global labels. You can add your own labels in the *Custom labels* tab.
+
+.Labels configuration screen
+image::web-labels-config_51.png[Labels configuration screen]
+
+.Procedure
+
+. In the {WebName}, click *Labels configuration*.
+. Click *Add label*.
+. To upload a labelset file, click the *Upload* tab, click *Browse*, select one or more files, and click *Close.*
++
+A labelset file must have an `.mta.label.xml` or `.windup.label.xml` extension. The uploaded file is stored on the {ProductShortName} server.
++
+. To register the server path of a labelset file, click the *Server path* tab, enter the *Labels* path, and click *Save*.
```

Comment:
Make sure that "click Save" is in all the server path steps!!!

Reply:
Strangely -- and perhaps this is a bug -- when you enter the server path for an app, no Save button appears.